master/buildbot/steps/source/p4.py

```diff
@@ -14,14 +14,14 @@
 # Copyright Buildbot Team Members
 # Portions Copyright 2013 Bad Dog Consulting
 import re
-from types import StringType
 
 from future.utils import string_types
 from twisted.internet import defer
 from twisted.python import log
 
 from buildbot import config
 from buildbot import interfaces
+from buildbot import util
```

Comment:
I hace updated the diff.
isort disagrees with you on this particular point about where util goes in the import list.